src/Connector.SqlServer/Features/IBuildDeleteDataFeature.cs

```diff
@@ -11,7 +12,8 @@ public interface IBuildDeleteDataFeature
     {
```

Comment:
Do not change a public interface.

There are three options here:

Go with a breaking change (Not allowed).
Default implementation (Not popular as it hides in the IDE's).
Extend the interface (I would do this, however any type checks would need to be updated).

Also SanitizedSqlName should be a string with the sanitization performed elsewhere.